modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -661,7 +670,7 @@ If set to `true`, the `providedCaFile` field is ignored.
 | Select the type of TLS configuration: +
 
 - `Disabled` (default) to not configure TLS for the endpoint.
-- `Provided` to manually provide cert file and a key file. Unsupported *.
+- `Provided` to manually provide cert file and a key file. [Unsupported (*)].
```

Comment:
You might want to check the preview here. This is rendering on the same line as `Disabled` and I'm not sure if that's intentional.